tendermint-rs/Cargo.toml

```diff
@@ -40,9 +40,9 @@ prost-amino-derive = { version = "0.4.0", optional = true }
 rand_os = { version = "0.1", optional = true }
 ring = { version = "0.14", optional = true }
 serde = { version = "1", optional = true, features = ["derive"] }
-serde_json = { version = "1", optional = true }
+serde_json = { version = "1"} 
 signatory = { version = "0.12", features = ["ed25519", "ecdsa"] }
-signatory-dalek = { version = "0.12", optional = true }
+signatory-dalek = { version = "0.12"}
```

Comment:
temporary. using it in a test to generate keys until I make proper test vectors. not sure how to use crates in tests when optional=true

Reply:
You can have the same crate in both `dependencies` and `dev-dependencies`, with the `optional = true`omitted in the latter